lib/resources/r4/encounter.yaml

```diff
@@ -547,6 +547,24 @@ fields:
         info_link: http://hl7.org/fhir/r4/v3/ObservationValue/cs.html
     note: Encounters created with meta.security set with code 'UNCERTREL' will be returned with an external encounter type.
 
+- name: Custom Attribute Value Extension
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute?_format=json
+  terminology_name: extension[x].extension[x].valueCodeableConcept
+  type: Extension
+  description: The value of the client defined custom attribute. This extension is nested under a <code>custom-attribute</code> extension.
+  action: terminology
+  binding:
+    description: The value of the client defined custom attribute. This binding is used for the <code>custom-attribute-value</code> extension nested under <code>customm-attribute</code> extensions.
+    terminology:
+      - display: Custom Attribute Value
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/&lt;codeSet&gt;
```

Comment:
the second codeSet should be
```
&lt;code set&gt;
```

Reply:
[updated](https://github.com/cerner/fhir.cerner.com/pull/438/commits/0c517a7db88c7ba427456f9aaf1e0805482a4527)